src/components/m-header-meta-right/_template.js

```diff
@@ -1,8 +1,10 @@
 import html from 'nanohtml';
 
-export default (props, { children }) => html`
+// IE11 does not support children on fragments
+// ref: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10060579/
+export default (props, { children, childNodes }) => html`
   <ul class="m-header-meta-right__list">
-    ${children && Array.from(children).map(child => html`
+    ${Array.from(children || childNodes).map(child => html`
```

Comment:
This logic should be in Abstarct not inside the template

Reply:
done